test/test_utils/__init__.py

```diff
@@ -1211,6 +1211,15 @@ def get_cuda_version_from_tag(image_uri):
     return cuda_framework_version
 
 
+def get_cuda_compare_string(cuda_version):
+    """
+    Return the cuda version string for comparison with packaging.version.Version
+    :param cuda_version: cuda version
+    :return: cuda version compare string
+    """
+    return cuda_version[-3:-1] + '.' +  cuda_version[-1:]
```

Comment:
nit: I believe this will only work for cuda 10 and above, and this won't work if we have cuda version like 11.10. I think a safer way to do this would be through regex.

The cuXXX tag in general is ambiguous - so maybe in the future we'll want to introduce a more clear cuda versioning tag. 

For now, I think we will be safest if we make the assumption that if there are more than 3 numbers in a row, the first two are the major version and the rest are minor version.

Reply:
yes, it'll work for cuda>=10 with upto 10 minor versions.. do you want to include cuda<10? or the case where there are more than 10 minor versions? I thought both these cases are not so likely, hence didnt include that..

Reply:
for addressing the latter, we can use `cuda_version[2:4] + '.' +  cuda_version[4:]`